frontend/cmd/cmd.go

```diff
@@ -161,6 +165,16 @@ func (opts *FrontendOpts) Run() error {
 	}
 	logger.Info(fmt.Sprintf("Application running in %s", opts.location))
 
+	// NOTE: In the future we may want to include "opts.location" as resource attribute.
+	shutdown, err := frontend.InstallOpenTelemetryTracer(ctx, logger)
+	if err != nil {
+		return err
+	}
+	defer func() {
+		if err := shutdown(ctx); err != nil {
+			logger.Error("otel-sdk shutdown failed", "error", err)
+		}
+	}()
```

Comment:
Not a requirement, but I wonder if this could be moved into `Frontend.Run`?
I'd like to keep `cmd.go` as simple as possible.

Reply:
Sure, it just needs to be initialized before the service starts.  I placed it on top of the `Frontend.Run` method.